opencti-platform/opencti-front/src/utils/filters/filtersUtils.tsx

```diff
@@ -340,6 +340,9 @@ export const buildFiltersAndOptionsForWidgets = (
       filterGroups: filters && isFilterGroupNotEmpty(filters) ? [filters] : [],
     };
   }
+  if (isKnowledgeRelationshipWidget) {
```

Comment:
add a unit test of buildFiltersAndOptionsForWidgets in filtersUtils.test.tsx (in an other PR maybe since the resolution of the issue is critical)